MapboxCoreNavigation/RouteController.swift

```diff
@@ -32,19 +32,23 @@ open class RouteController: NSObject {
     */
     public var snapsUserLocationAnnotationToRoute = false
     
+    var simulate: Bool = false
```

Comment:
Properties shouldn’t be verbs, because that causes them to look like methods. How about `simulatesLocationUpdates`?